pkg/arvo/gen/validate-ship.hoon

```diff
@@ -0,0 +1,108 @@
+::  Validate userspace state
+::
+::  /hoon/validate-ship/gen
+::
+/+  resource
+:-  %say
+|=  $:  [now=@da eny=@uvJ =beak]
+        [~ ~]
+    ==
+:-  %noun
+=|  issues=(list tape)
+|^
+=.  issues
+  check-group-syncs
+issues
+::
+++  scry-synced
```

Comment:
Metadata and contacts should at some point be moved onto `push-hook` and `pull-hook` (and away from `path` and onto `resource` identifiers) as well... But yeah, not related to this PR